main_js.go

```diff
@@ -704,17 +705,20 @@ func (pe *pcdeditor) runImpl(ctx context.Context) error {
 		//   - in the crop box
 		//   - in the select box
 		//   - close to the mouse cursor position given as (x, y)
-		scanSelection := func(x, y int) ([]uint32, bool) {
+		scanSelection := func(x, y int) bool {
 			if hasPointCloud && pp.Points > 0 {
 				origin, dir := perspectiveOriginDir(x, y, width, height, &projectionMatrix, &modelViewMatrix)
 
 				// Run GPGPU shader
 				gl.UseProgram(programComputeSelect)
-				clean := enableVertexAttribs(gl, aVertexPosition)
+				clean := enableVertexAttribs(gl, aVertexPosition, aSelectMask)
 				defer clean()
 
 				gl.BindBuffer(gl.ARRAY_BUFFER, posBuf)
 				gl.VertexAttribPointer(aVertexPosition, 3, gl.FLOAT, false, pp.Stride(), 0)
+				gl.BindBuffer(gl.ARRAY_BUFFER, selectMaskBuf)
+				gl.VertexAttribIPointer(aSelectMask, 1, gl.UNSIGNED_INT, 4, 0)
```

Comment:
Point select mask buffer was cleared by `scanSelection`. Call of `scanSelection` was suppressed on single key Delete and Label commands if select mode is Segment, but wasn't on command-line commands.

This PR makes SegmentSelect bit of point select mask buffer uncleared by `scanSelection`.